definitions.md

```diff
@@ -251,9 +251,13 @@ In the context of:
 
 A `ticket` is used as a source of randomness in EC leader election. Every block depends on an `ElectionProof` derived from a `ticket`. At least one new `ticket` is produced with every new block. Ticket creation is described [here](./expected-consensus.md#Ticket-generation).
 
+#### Ticket Chain
+
+Each blockchain in Filecoin can be associated to a given `ticket chain`. The `ticket chain` is assembled by taking the tickets (usually one) contained by the block with the smallest final ticket in each of the chain's `TipSet`s.
```

Comment:
each block or each block chain?

Reply:
each chain I guess, so blockchain but that's an overloaded term